set.mm

```diff
@@ -634895,6 +634910,264 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       EKUMZAFVIUNTTUO $.
   $}
 
+  ${
+    $d A y $.  $d I x $.  $d I y $.  $d M x $.  $d M y $.  $d X y $.
+    $d ph x $.  $d ph y $.
+    metakunt5.1 $e |- ( ph -> M e. NN ) $.
+    metakunt5.2 $e |- ( ph -> I e. NN ) $.
+    metakunt5.3 $e |- ( ph -> I <_ M ) $.
+    metakunt5.4 $e |- A = ( x e. ( 1 ... M ) |-> if (
+       x = I , M , if ( x < I , x , ( x - 1 ) ) ) ) $.
+    metakunt5.5 $e |- C = ( y e. ( 1 ... M ) |-> if (
+       y = M , I , if ( y < I , y , ( y + 1 ) ) ) ) $.
+    metakunt5.6 $e |- ( ph -> X e. ( 1 ... M ) ) $.
+    $( C and A are inverse on one side.  (Contributed by metakunt,
```

Comment:
This theorem is perhaps better understood, if you explicitly say that A is a primitive permutation that moves the I-th element to the end, and C moves the last element to the I-th position.

Reply:
No I've meant right and left inverse (section, retraction) but I always switch the names so I just said one sided inverse so that I don't write the wrong names. Fixed the description of metakunt14 and clarified right/left inverse on the correct places.